macrosynergy/visuals/README.md

```diff
@@ -0,0 +1,186 @@
+# macrosynergy.visuals
+
+## Description
+
+The `macrosynergy.visuals` subpackage contains functions for visualizing quantamental data, and to provide `view.*` functions for the `macrosynergy` package.
+
+The functionality is built around the ability to create quick generic plots of data, and to provide a framework for creating custom plots.
+Simplicity, speed, consistency and flexibility are the main goals.
+The core visualization functions are built around `matplotlib`, with goals of mimicking the rich and dataframe-friendly nature of `seaborn`.
+
+## Functionality
+
+The `macrosynergy.visuals` subpackage features the following functionality:
+
+### User-facing functions:
+
+- `view.*` functions: A set of preset methods to view data in commonly used contexts.
+
+  - `timelines()`: A function for viewing time-series data. Uses `FacetPlot.lineplot()` and `LinePlot` internally.
+  - `availability()`: A function for viewing data availability. Uses `Heatmap` internally.
+  - `correlation()`: A function for viewing correlation matrices. Uses `Heatmap` internally.
+  - `metrics()`: A function for viewing metrics. Uses `Heatmap` internally.
+  - `ranges()`: A function for viewing ranges. Uses `BoxPlot` and `BarPlot` internally.
+  - `distribution()`: A function for viewing distributions. Uses `FacetPlot.boxplot()` and `BoxPlot` internally.
+  - `reg_scatter()`: A function for viewing scatter plots. Uses `FacetPlot.scatterplot()` and `ScatterPlot` internally.
```

Comment:
Instead of describing what a function uses internally, describe behaviour and expected outcome from using the function. Back-end can change but input/output should stay stable.

Reply:
Will change soon